tests/Cms/Helpers/HelpersTest.php

```diff
@@ -144,6 +144,27 @@ public function testHandleErrorsException()
 		);
 	}
 
+	/**
+	 * @covers ::handleErrors
+	 */
+	public function testHandleErrorsLocaleReset()
+	{
+		setlocale(LC_ALL, 'de_DE.UTF-8');
```

Comment:
`Suggestions`:
- As this changes global state, we should backup the original locale in `setUp()` and restore it in `tearDown()`
- I think it's enough to set `LC_MESSAGES` to keep the change to global state as minimal as possible

Reply:
Added these.